uportal-war/src/main/webapp/WEB-INF/web.xml

```diff
@@ -414,6 +414,8 @@
 
     <servlet-mapping>
         <servlet-name>JsonRenderingDispatcherServlet</servlet-name>
+        <url-pattern>/layout/v2/layout.json</url-pattern>
+        <url-pattern>/layout/v1/layout.json</url-pattern>
```

Comment:
I think the next version (and subsequent versions) of this API should simply be moved under /api so that we don't have to keep managing servlet-mappings in web.xml.